emission/integrationTests/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get install -y curl
 RUN echo "About to copy e-mission server code"
 COPY start_integration_tests.sh/ /start_integration_tests.sh
 
-ENV NOMINATIM_QUERY_URL='nominatim:8080'
```

Comment:
is there a reason you put the `ENV` in the Dockerfile instead of the docker-compose?
Think about compile-time and run-time settings and how they map to image build and image run...